c2corg_ui/static/js/editing/routeediting.js

```diff
@@ -34,6 +34,18 @@ app.RouteEditingController = function($scope, $element, $attrs, $http,
     $uibModal, $compile, appLang, appAuthentication, ngeoLocation, appAlerts,
     appApi, authUrl, appDocument, appUrl, imageUrl) {
 
+  /**
+   * @type {Object}
+   * @private
+   */
+  this.modal__ = $uibModal;
```

Comment:
Why is the underscore doubled? Only one should be enough as for the other private properties.

I think you could reuse the existing ``this.modal_`` property that is defined in the parent controller:
https://github.com/c2corg/v6_ui/blob/master/c2corg_ui/static/js/editing/documentediting.js#L164-L174

But you have to make it ``@public`` and rename it to ``this.modal``.

See ``this.auth`` https://github.com/c2corg/v6_ui/blob/master/c2corg_ui/static/js/editing/documentediting.js#L85-L89